### PR TITLE
tools: Add page for gccrs

### PR DESCRIPTION
**Pull request by CohenArthur**
Add a small introduction to gccrs and useful links for following the project. This commit also modifies SUMMARY.md to link to the gccrs page.

Thank you Miguel for the opportunity :)

Pinging @philberty for review

### CODE REVIEW

**Comment by philberty** [comment]
Looks good to me!

File: src/gccrs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+[`gccrs`](https://github.com/rust-gcc/gccrs) is an alternative compiler for the Rust
+programming language, developed from the ground up for the
+[GNU Compiler Collection(GCC)](https://gcc.gnu.org/) project.
+
+While `gccrs` is not yet able to compile Rust code, it is progressing fast - we are hoping we will be able to compile the Rust 1.49 standard library by the next GCC release, 14.1.
+
+Once that milestone is achieved, we will begin catching up to the Rust version used by Rust-for-Linux - in the hopes of being useful quickly!
+
+You can follow the project's progress on [our blogpost](https://rust-gcc.github.io)
```
Comment (ojeda) [inline src/gccrs.md:11]:
```suggestion
You can follow the project's progress on [our blogpost](https://rust-gcc.github.io).
```

File: src/gccrs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# `gccrs`
+
+[`gccrs`](https://github.com/rust-gcc/gccrs) is an alternative compiler for the Rust
+programming language, developed from the ground up for the
+[GNU Compiler Collection(GCC)](https://gcc.gnu.org/) project.
```
Comment (ojeda) [inline src/gccrs.md:5]:
```suggestion
[GNU Compiler Collection (GCC)](https://gcc.gnu.org) project.
```

File: src/gccrs.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+While `gccrs` is not yet able to compile Rust code, it is progressing fast - we are hoping we will be able to compile the Rust 1.49 standard library by the next GCC release, 14.1.
+
+Once that milestone is achieved, we will begin catching up to the Rust version used by Rust-for-Linux - in the hopes of being useful quickly!
```
Comment (ojeda) [inline src/gccrs.md:9]:
```suggestion
While `gccrs` is not yet able to compile Rust code, it is progressing fast — we are hoping we will be able to compile the Rust 1.49 standard library by the next GCC release, 14.1.

Once that milestone is achieved, we will begin catching up to the Rust version used by Rust-for-Linux — in the hopes of being useful quickly!
```

**Review by ojeda — COMMENTED** [review]
Thanks a ton guys!

A few suggestions -- all up to you, please feel free to ignore.

**Comment by CohenArthur** [comment]
thanks for the review @ojeda :D all good suggestions, they've all been applied

File: src/gccrs.md (hunk 1 above)
Comment (ojeda) [reply src/gccrs.md]: This one had a slightly hard to see period suggestion at the end, not sure if you noticed.

**Comment by ojeda** [comment]
Thanks Arthur!

**Comment by ojeda** [comment]
It is live now! :)

https://rust-for-linux.com/gccrs

(I added the period assuming you didn't see it to avoid making you force push again, but please let me know if you didn't want it. I also linked the new page from our Contributing page -- previously I was linking to your blog: https://github.com/Rust-for-Linux/rust-for-linux.com/commit/4864d530b26d26404570b603f82b14548383acca)

**Comment by ojeda** [comment]
My pleasure!

**Pull request merged**